### PR TITLE
Enable collection configuration through env var

### LINKED ISSUE #200: Enable collection configuration through env var for gisaia/arlas-wui
Docker image `gisaia/arlas-wui` may support `ARLAS_SERVER_COLLECTION` environment variable to inject `arlas.server.collection` property of `config.json` at runtime, like `ARLAS_SERVER_URL` already does for ARLAS-server url config.

Documentation would have to be updated : http://docs.arlas.io/arlas-tech/current/arlas-wui-configuration/

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
- Fix #200

### CODE REVIEW

File: scripts/start.sh [hunk 1]
```diff
@@ -60,10 +60,19 @@ fi
...
   ARLAS_SERVER_URL="http://demo.arlas.io/arlas/"
   export ARLAS_SERVER_URL
-  echo "The default ARLAS-server url is used"
+  echo "The default ARLAS-server url '${ARLAS_SERVER_URL}' is used"
 else
   echo ${ARLAS_SERVER_URL}  "is used as 'arlas.server.url'"
 fi
+
+if [ -z "${ARLAS_SERVER_COLLECTION}" ]; then
+  ARLAS_SERVER_COLLECTION="demo_ais-danmark"
```
Comment (laurent-thiebaud-gisaia) [inline scripts/start.sh:69]: In find strange, in an open source project, to reference one of our projects

Comment (MohamedHamouGisaia) [reply scripts/start.sh:69]: ARLAS-wui needs ARLAS-server to fetch the data

Comment (sfalquier) [reply scripts/start.sh:69]: Maybe something like `foobar` ou `lorem-ipsum`

Comment (laurent-thiebaud-gisaia) [reply scripts/start.sh:69]: Sorry, the reference to demo_ais-danmark is strange, not arlas-server

**Pull request merged**